### PR TITLE
Add more telemetry data

### PR DESCRIPTION
**Pull request by nevalla**
Signed-off-by: Lauri Nevala <lauri.nevala@gmail.com>

### CODE REVIEW

File: src/extensions/core-api/stores.ts [hunk 1]
```diff
@@ -1 +1,3 @@
 export { ExtensionStore } from "../extension-store"
+export { clusterStore, ClusterModel } from "../../common/cluster-store"
```
Comment (nevalla) [inline src/extensions/core-api/stores.ts:2]: What type we should export for `Cluster`?

**Review by jakolehm — APPROVED** [review]

File: src/extensions/core-api/stores.ts (hunk 1 above)
Comment (nevalla) [reply src/extensions/core-api/stores.ts:2]: `Cluster` type is now also exported, so maybe use it here too

File: src/extensions/core-extension-api.ts [hunk 2]
```diff
@@ -7,10 +7,12 @@ export * from "./lens-renderer-extension"
 import * as EventBus from "./core-api/event-bus"
 import * as Store from "./core-api/stores"
 import * as Util from "./core-api/utils"
+import * as App from "./core-api/app"
```
Comment (ixrock) [inline src/extensions/core-extension-api.ts]: BTW this file is actually moved to `extensions/core-api/index.ts`
But it seems it's still exists in base branch (removed in follow up PR: https://github.com/lensapp/lens/pull/1140)

Comment (nevalla) [reply src/extensions/core-extension-api.ts]: fixed and removed also here

**Pull request merged**